.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     working_directory: ~/openreview-expertise
     docker:
       - image: ubuntu:18.04
-      - image: circleci/redis:6.0.0
+      - image: redislabs/redisai
```

Comment:
Does this also start a regular instance of Redis/would there be an issue with keeping both images in CircleCI? I'm wondering because my Redis PR does not use redisai

Reply:
Yes, `redisai` image launches regular instance of Redis as well, we don't need the circle ci redis.